uberfire-extensions/uberfire-security/uberfire-security-management/uberfire-security-management-keycloak/src/main/java/org/uberfire/ext/security/management/keycloak/KeyCloakGroupManager.java

```diff
@@ -104,9 +104,10 @@ public Group get(String identifier) throws SecurityManagementException {
     @Override
     public List<Group> getAll() throws SecurityManagementException {
         final List<Group> roles = new LinkedList<>();
+        final RolesResource[] rolesResource = new RolesResource[1];
```

Comment:
Why there is a list of 1 element here? Can't we wait until the list is needed to create a `Collection.singletonList()`?

Reply:
@adrielparedes you are right we dont need a list here. But  in method "Group get(String identifier)", i remember it was added to intialise the interface RolesResource since it was used inside and outside lambda expression. Please see

Reply:
Got it!